sphinx/cmd/quickstart.py

```diff
@@ -138,25 +138,25 @@ def ok(x):
 
 
 def term_decode(text):
-    # type: (unicode) -> unicode
+    # type: (Union[bytes,unicode]) -> unicode
```

Comment:
Please use `str` instead, because we use py2 types to represent typing.

Reply:
`bytes` is a py2 type too though

Reply:
I know, but it is an alias of `str` on py2.
```
>>> str
<type 'str'>
>>> bytes
<type 'str'>
```

Reply:
Indeed. I'd argue that `bytes` is clearer, especially for people who don't know which version of python you run mypy with.

But if you insist, I'll change this.

Reply:
Oh, sorry. `bytes` is already used in our project.
```
$ grep -r type: sphinx | grep bytes
sphinx/io.py:        # type: (Union[unicode, bytes]) -> unicode
sphinx/util/inventory.py:        # type: () -> Iterator[bytes]
```
So no reason to change this. Sorry for trivial things.

Reply:
Thanks for checking - I hadn't actually checked which you used myself!